docs/cli-tool.md

```diff
@@ -0,0 +1,89 @@
+# Yari's CLI Tool
+
+## Basic usage
+
+Install dependencies:
+
+```sh
+yarn
+```
+
+Run the CLI tool:
+
+```sh
+yarn tool --help
+
+  tool 0.0.0
+
+  USAGE
+
+    ▸ yarn tool <command> [ARGUMENTS...] [OPTIONS...]
+
+
+  COMMANDS — Type 'yarn tool help <command>' to get some help about a command
+
+    validate-redirect                    Check the _redirects.txt file(s)
+    test-redirect                        Test a URL (pathname) to see if it
+                                         redirects
+    add-redirect                         Add a new redirect
+    delete                               Delete content
+    move                                 Move content to a new slug
+    edit                                 Spawn your EDITOR for an existing slug
+    create                               Spawn your Editor for a new slug
+    validate                             Validate a document
+    preview                              Open a preview of a slug
+
+  GLOBAL OPTIONS
+
+    -h, --help                           Display global help or command-related
+                                         help.
+    -V, --version                        Display version.
+    --no-color                           Disable use of colors in output.
+    -v, --verbose                        Verbose mode: will also output debug
+                                         messages.
+    --quiet                              Quiet mode - only displays warn and error
+                                         messages.
```

Comment:
That's just bound to become out of date :)
Perhaps it's enough to say:

    Run the CLI tool:

    ```sh
    yarn tool --help
    ```